src/web/components/SiteTour/tourStorage.spec.ts

```diff
@@ -61,4 +76,38 @@ describe('Tour tests', () => {
     expect(versions[0]).toBe('0.0.0');
     expect(versions.slice(-1)[0]).toBe('102.102.102');
   });
+
+  test('all steps shown for those with the correct user roles and multiple participants', () => {
```

Comment:
nit: this test doesn't strictly test that the steps are _shown_, rather, that they are returned as a step from `GetTourSteps`. Perhaps we should adjust the name of the test to differentiate between that and any UI rendering/functionality.